live/global.ini

```diff
@@ -44090,7 +44090,7 @@ dfm_ui_Throttle_Pad=Um die Leistung der Haupttriebwerke zu erhöhen, können Sie
 dfm_ui_Throttle_Title=Throttle
 dfm_ui_TimeRemainingInMatch=VERBLEIBENDE ZEIT IM SPIEL
 dfm_ui_TimeToCruiseMax=Maximale Kreuzfahrtzeit:
-dfm_ui_Time_To_Respawn_msg=Respawn in %i Sekunden möglich
+dfm_ui_Time_To_Respawn_msg=RESPAWN IN %i SEKUNDEN
```

Comment:
Darüber könnte man streiten. Nicht in allen Modi respawnst du ohne etwas zu drücken. Also ist es erst dann "Möglich".

Reply:
Ok. Habe es korrigiert :)